integration-tests-ex/cases/src/test/java/org/apache/druid/testsEx/config/ClusterConfigTest.java

```diff
@@ -85,15 +79,6 @@ public void testYaml() throws FileNotFoundException
     assertEquals("http://localhost:8888", service.clientUrl());
     assertEquals("http://localhost:8888", resolved.routerUrl());
 
```

Comment:
Can I ask you to comment this out and not remove it? The functionality needs to be tested; let's not use the code itself.

Reply:
Will make this change in the next PR, which will a fix for the batch index test which were disabled here.
Would not like to disturb the green build for this change.